cli/commands/deploy/index.js

```diff
@@ -1,75 +1,129 @@
+      .replace(/{{branchName}}/g, branchName)
+      .replace(/{{repoUrl}}/g, repoUrl);
+
+    fs.writeFileSync(appYamlPath, appYamlContentFilled);
+    fs.writeFileSync(deployTemplateYamlPath, deployTemplateYamlContentFilled);
 
-    const { platform } = await inquirer.prompt({
-        type: "list",
-        name: "platform",
-        message: "Select the deployment platform",
-        choices: ["Digital Ocean", "AWS"]
-    });
-
-    if (platform === "Digital Ocean") {
-        const { branchName, repoUrl } = getGitInfo();
-
-        const projectRootPath = process.cwd();
-        const deployFolderPath = path.join(projectRootPath, '.do');
-
-        if (fs.existsSync(deployFolderPath)) {
-            console.log("Deployment .do directory already exists. Nothing to do.");
-            return;
-        }
-        
-        const appYamlPath = path.join(deployFolderPath, 'app.yaml');
-        const deployTemplateYamlPath = path.join(deployFolderPath, 'deploy.template.yaml');
-
-        if (!fs.existsSync(deployFolderPath)) {
-            fs.mkdirSync(deployFolderPath);
-        }
-
-        const appYamlContentFilled = appYamlContent
-            .replace(/{{projectName}}/g, projectName)
-            .replace(/{{branchName}}/g, branchName)
-            .replace(/{{repoUrl}}/g, repoUrl);
-
-        const deployTemplateYamlContentFilled = deployTemplateYamlContent
-            .replace(/{{projectName}}/g, projectName)
-            .replace(/{{branchName}}/g, branchName)
-            .replace(/{{repoUrl}}/g, repoUrl);
-
-        fs.writeFileSync(appYamlPath, appYamlContentFilled);
-        fs.writeFileSync(deployTemplateYamlPath, deployTemplateYamlContentFilled);
-
-        console.log(successMessage);
-        console.log(detailsMessage);
-    } else if (platform === "AWS") {
-        console.log("AWS deployment coming soon.");
+    console.log(successMessage);
+    console.log(detailsMessage);
+
+  } else if (platform === "AWS") {
+    const cdkJsonPath = path.join(projectRootPath, 'cdk.json');
+    const circleciFolderPath = path.join(projectRootPath, '.circleci');
+    const configYmlPath = path.join(circleciFolderPath, 'config.yml');
+    const stackFolderPath = path.join(projectRootPath, 'stack');
+
+    if (
+      fs.existsSync(cdkJsonPath) &&
+      fs.existsSync(configYmlPath) &&
+      fs.existsSync(stackFolderPath) &&
+      fs.existsSync(path.join(stackFolderPath, 'index.ts')) &&
+      fs.existsSync(path.join(stackFolderPath, 'rds.ts')) &&
+      fs.existsSync(path.join(stackFolderPath, 'vpc.ts')) &&
+      fs.existsSync(path.join(stackFolderPath, 'elastic-beanstalk.ts')) &&
+      fs.existsSync(path.join(stackFolderPath, 'variables.ts'))
+    ) {
+      console.log(chalk.yellow`AWS deployment setup already exists. ${chalk.cyan("Nothing to do.")} `);
+      return;
+    }
+
+    console.log("Installing AWS CDK packages...");
+    execSync("npm install aws-cdk-lib constructs @types/node -D", { stdio: "inherit" });
```

Comment:
we need to set the working directory to `project root path` when doing this operation